05LessonFive/LessonFive.js

```diff
@@ -15,6 +25,21 @@ module.exports = function() {
     // divisible by 3 are replaced by the string "fizz", numbers divisible by 5
     // are replaced by the string "buzz", and the numbers divisble by both 3 and
     // 5 are replaced by fizzbuzz
-
+    var arr=[];
+    for(var i=0; i<100; i++) {
+      if(i%15===0) {
+        arr.push('fizzbuzz');
+      }
+      else if(i%3===0) {
+        arr.push('fizz');
+      }
+      else if(i%5===0) {
+        arr.push('buzz');
+      }
+      else {
+        arr.push(i);
+      }
+    }
+    return arr;
   }
```

Comment:
Good work here! I'm curious if you can defend the usage of 

```js
if(i%15===0){...
```

over

```js
if(i%5===0 && i%3===0) {
```

Do you know why `i%15===0` works?